l10n_es_aeat/security/aeat_security.xml

```diff
@@ -15,6 +15,6 @@
         <field eval="True" name="global" />
         <field
             name="domain_force"
-        >['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
+        >['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]</field>
```

Comment:
Esto require script de migración:

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration

Reply:
eso si fuera noupdate=1, pero es noupdate=0, por lo que no creo que haga falta

Reply:
Vale, pues eso es un error a priori, porque debería ser noupdate="1". Cámbialo si no y script no hace falta, aunque todos los migrados seguirán con noupdate=0